contracts/deploy/005_compensation_claims.js

```diff
@@ -32,6 +23,8 @@ const compensationClaimsDeploy = async ({ getNamedAccounts, deployments }) => {
   const OUSD = await ethers.getContract("OUSDProxy");
   log(`Using OUSD address ${OUSD.address}`);
 
+  await deployWithConfirmation("Timelock", [governorAddr, 60]);
```

Comment:
I don't think we need this deploy. This contract can be owned by the Governor contract which is already deployed.